### PR TITLE
Added my customized 40% layout

### PR DESCRIPTION
**Pull request by alper**
## Description

I had to heavily customize the default 40% cartel layout to be able to use it on macOS Dvorak so I thought I'd publish it here for others to reuse.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

### CODE REVIEW

**Comment by alper** [comment]
All the other contra keymaps seem to not build?

**Comment by ridingqwerty** [comment]
> All the other contra keymaps seem to not build?

From the travis build logs, only the "losinggeneration", "msiu", and "xyverz" keymaps failed to build, which bomb on my fork as well.  Everything else seemed to build just fine.

File: keyboards/contra/keymaps/alper/keymap.c [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
```
Comment (noroadsleft) [inline keyboards/contra/keymaps/alper/keymap.c]: The `QMK_KEYBOARD_H` include above does this automatically; you don't need this line here.

```suggestion
```

File: keyboards/contra/keymaps/alper/keymap.c [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+  _ADJUST
+};
+
+enum planck_keycodes {
+  QWERTY = SAFE_RANGE,
+  COLEMAK,
+  DVORAK,
+  LOWER,
+  RAISE,
+  BACKLIT
```
Comment (noroadsleft) [inline keyboards/contra/keymaps/alper/keymap.c]: This keycode isn't being used.

```suggestion
```

File: keyboards/contra/keymaps/alper/keymap.c [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+        #ifdef BACKLIGHT_ENABLE
+          backlight_step();
+        #endif
+        PORTE &= ~(1<<6);
+      } else {
+        unregister_code(KC_RSFT);
+        PORTE |= (1<<6);
+      }
+      return false;
+      break;
```
Comment (noroadsleft) [inline keyboards/contra/keymaps/alper/keymap.c]: Lines 192-204 can all be deleted.

**Comment by noroadsleft** [comment]
@ridingqwerty is correct; for this PR, as long as your keymap compiles, it's fine. Any other failing keymaps would be out of scope.

**Comment by alper** [comment]
Comments resolved.

**Comment by drashna** [comment]
For future reference, we recommend against committing to your `master` branch as you've done here, because pull requests from modified `master` branches can make it more difficult to keep your QMK fork updated. It is highly recommended for QMK development – regardless of what is being done or where – to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

There are instructions on how to keep your fork updated here:

[**Best Practices: Your Fork's Master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_best_practices?id=your-fork39s-master-update-often-commit-never)

If you need any help with this just ask.

--- Courtesy of @noroadsleft 

And to emphasize, we're not saying you did anything wrong.  This is more of "so you can make things easier on yourself by using what we've learned from our mistakes"

**Comment by alper** [comment]
Thanks for the tip, juggling `origin` and `upstream` is always a bit of a hell and you _do not_ want to know how much I messed up my branches before I could create this Pull Request.

**Pull request merged**